tools/executors/directus/impl.js

```diff
@@ -0,0 +1,198 @@
+'use strict'
```

Comment:
Compiled version of the Directus executor. Should always be in source control to avoid having to re-compile it locally.